gpii/node_modules/solutionsRegistry/src/js/filtered-settings-middleware.js

```diff
@@ -0,0 +1,174 @@
+/* eslint-env node */
+"use strict";
+var fluid = require("infusion");
+var gpii  = fluid.registerNamespace("gpii");
+
+fluid.require("%kettle");
+fluid.require("%gpii-json-schema");
+
+// Add the ability to require JSON5 files.
+require("json5/lib/register");
+
+// Require universal so that we can resolve paths to it and any sub-modules.
+require("../../../../../");
```

Comment:
fluid.require("%gpii-universal") is clearer. We tend to try to avoid requires with leading ".." as fragility risks.

Reply:
Done, throughout.